platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -92,7 +92,7 @@ public record PlatformSchedulersConfig(
                 TaskSchedulerConfiguration stateSignatureCollector,
         @ConfigProperty(
                         defaultValue =
-                                "SEQUENTIAL_THREAD CAPACITY(30) FLUSHABLE SQUELCHABLE UNHANDLED_TASK_METRIC BUSY_FRACTION_METRIC")
+                                "SEQUENTIAL_THREAD CAPACITY(100000) FLUSHABLE SQUELCHABLE UNHANDLED_TASK_METRIC BUSY_FRACTION_METRIC")
```

Comment:
If we assume 10k TPS and ~1 round per second, this is roughly 3x less capacity than before. Not necessarily a problem, but I advise testing on a large network prior to merging (as opposed to just using a one node benchmark). In the early days of testing the health monitor, I observed the transaction handling thread not being fully utilized when this specific scheduler had a capacity that was too low.